### DISCUSSION TITLE
Support for Compose Multiplatform?

### DISCUSSION THREAD

**Discussion by oxyroid** [bumble-tech/appyx#260] [Ideas]
Will appyx support for Compose Multiplatform?

**Comment by zsoltk** [comment]
Hi @thxbrop! We like the idea too, and have made a successful experiment with it. There are some considerations why we kept it open at this point, but we'd like to get back to it.

**Comment by marcguilera** [comment]
Looks like there is some interest in this. What are the considerations? 

I could give it a try. I think possibly using multiplatform uuid and something like the essenty lifecycle and back handler (or a custom one) the rest is pretty much multiplatform ready. I could try to give it a spin unless you think it's better to wait for an _official_ solution.

**Comment by zsoltk** [comment]
Hey @realOxy @marcguilera our 2.0-alpha01 is out now. In the 2.x (now default) branch the library is split to navigation + interactions part, as you can check on: https://bumble-tech.github.io/appyx/ 

Interactions is already compose-multiplatform.
Navigation is becoming multiplatform too, and it's close to being merged.
